packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -128,6 +132,12 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 		} ),
 	] );
 
+	if ( blockAPIVersion < 2 ) {
+		warning(
```

Comment:
Should we error or warn?

Reply:
`warning` is our custom helper that works nearly the same as React warnings that get applied only for development builds.